web/client-api/src/main/java/io/deephaven/web/client/api/i18n/JsTimeZone.java

```diff
@@ -12,6 +12,28 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Represents the timezones supported by Deephaven. Can be used to format dates, taking into account the offset changing
+ * throughout the year (potentially changing each year). These instances mostly are useful at this time to pass to the
+ * `DateTimeFormat.format()` methods, though also support a few properties at this time to see details about each
+ * instance.
+ *
+ *
+ * The following timezone codes are supported when getting a timezone object - instances appearing in the same line will
+ * return the same details:
+ *
+ *
+ * - `GMT`/`UTC` - `Asia/Tokyo` - `Asia/Seoul` - `Asia/Hong_Kong` - `Asia/Singapore` - `Asia/Calcutta`/`Asia/Kolkata` -
```

Comment:
Probably should explicitly break this into ul/li items, since the spotless pass is murdering it